src/Documents/Subscriptions/SubscriptionWorker.ts

```diff
@@ -314,9 +314,12 @@ export class SubscriptionWorker<T extends object> implements IDisposable {
                     + " cannot be opened, because it's in use and the connection strategy is "
                     + this._options.strategy);
             case "Closed":
-                throwError("SubscriptionClosedException",
+                const canReconnect = connectionStatus.data.CanReconnect || false;
+                const subscriptionClosedError = getError("SubscriptionClosedException",
                     "Subscription with id " + this._options.subscriptionName
                     + " was closed. " + connectionStatus.exception);
+                (subscriptionClosedError as any).canReconnect = canReconnect;
```

Comment:
In line 317 it's `CanReconnect`. Here it's `canReconnect`. Is that the intent?

Reply:
yes - on the wire we have upper case - on model we want tohave lower case